src/app/metadata-schema-form/metadata-form/metadata-form.component.html

```diff
@@ -58,13 +58,23 @@
             Reset
           </a>
 
-          <button *ngIf="selectedTabKey !== visibleTabs[visibleTabs.length - 1].key"
-            class="next-button vf-button vf-button--primary vf-button vf-u-margin__left--xxl">Next
-          </button>
-          <button *ngIf="selectedTabKey === visibleTabs[visibleTabs.length - 1].key"
-                  class="next-button vf-button vf-button--primary vf-button">
-            {{config.submitButtonLabel ? config.submitButtonLabel : 'Finish'}}
+          <button
+            *ngIf="selectedTabKey !== visibleTabs[visibleTabs.length - 1].key"
+            (click)="onSubmit($event)"
+            class="next-button vf-button--primary vf-button vf-u-margin__left--xxl">Next
           </button>
+          <ng-container *ngIf="selectedTabKey === visibleTabs[visibleTabs.length - 1].key">
+            <button class="next-button vf-button--primary vf-button" (click)="onSubmit($event)">
+              {{config.submitButtonLabel ? config.submitButtonLabel : 'Finish'}}
+            </button>
+            <button
+              *ngIf="allowValidationOverride"
+              class="next-button vf-button--tertiary vf-button"
+              (click)="onSubmit($event, true)"
+            >
+              {{config.submitButtonLabel ? config.submitButtonLabel : 'Finish'}} with Errors
```

Comment:
Making this behaviour generic in the metadata form component seems strange to me. 🤔  Alternatively, we could show a confirm box in the project metadata form which will let only a wrangler proceed even with errors.

Reply:
I feel like having a confirm dialogue box is more intuitive than suddenly displaying a button which says they can proceed with errors.

Reply:
I imagine the logic in adding a confirm dialogue box if the user is a wrangler can be placed here: https://github.com/ebi-ait/ingest-ui/blob/dev/src/app/projects/components/project-metadata-form/project-metadata-form.component.ts#L138-L146

If the wrangler says yes, it'll proceed to save the project, if no, it'll just do nothing and the wrangler could just check and fix the errors or try saving again. It's technically the same flow just that it uses a dialogue box instead of "save button with errors"

Reply:
Okay, done! Please see screenshot above